nxt_editor/integration/blender/__init__.py

```diff
@@ -23,12 +23,19 @@ def __init__(self):
         if b_major != 2 or b_minor < 80:
             raise RuntimeError('Blender version is not compatible with this '
                                'version of nxt.')
-        user_dir = os.path.expanduser('~/AppData/Roaming/Blender '
-                                      'Foundation/Blender/'
-                                      '{}.{}'.format(b_major, b_minor))
-        self.user_dir = user_dir
-        nxt_modules = os.path.join(user_dir, 'scripts/addons/modules')
-        self.modules_dir = nxt_modules.replace(os.sep, '/')
+
+        if 'darwin' in sys.platform:
+            user_dir = os.path.expandvars('/Users/$USER/Library/Application Support/Blender/{}.{}/')
+        elif 'linux' in sys.platform:
+            user_dir = os.path.expandvars('$HOME/.config/blender/{}.{}/')
+        elif 'win' in sys.platform:
+            user_dir = os.path.expanduser('~/AppData/Roaming/Blender '
+                                          'Foundation/Blender/'
+                                          '{}.{}')
+        else:
+            raise SystemError('Unsupported operating system!')
```

Comment:
Would `bpy.utils.user_resource('SCRIPTS', 'addons')` get you what you need?
https://docs.blender.org/api/current/bpy.utils.html#bpy.utils.user_resource
https://blender.stackexchange.com/a/20851

Reply:
Oh true! 🤦🏻